dist/index.html

```diff
@@ -60,7 +60,7 @@
         license: '3-Clause BSD License',
         labels: ['visualization', 'itk', 'vtk', 'image', 'mesh', 'geometry', 'point set', '2D', '3D'],
         icon: 'https://kitware.github.io/itk-vtk-viewer/app/favicon-32x32.png',
-        cover: ['https://kitware.github.io/itk-vtk-viewer/docs/howToUse.jpg'],
+        cover: ['https://kitware.github.io/itk-vtk-viewer/docs/howToUse.jpg', 'https://kitware.github.io/itk-vtk-viewer/docs/imjoy.png'],
```

Comment:
Good!

Reply:
It seems we have some display issue in imjoy.io: https://github.com/imjoy-team/ImJoy/issues/394